src/crypto/asmcrypto.js

```diff
@@ -0,0 +1,3 @@
+//# sourceMappingURL=asmcrypto.js.map
```

Comment:
Since asmCrypto currently does not have an npm release I asked the author to publish via npm in https://github.com/vibornoff/asmcrypto.js/issues/95. In the meantime I created a fork under the openpgpjs org with a branch that builds a minimal library (AES-CFB, SHA-1, SHA-256) specifically for use in openpgp.js:

https://github.com/openpgpjs/asmcrypto.js/compare/master...openpgpjs:openpgp_build

I then included asmCrypto in https://github.com/openpgpjs/openpgpjs/commit/cfe058596fc8d281ab57da254719b2f311c1cc3e and rusha in https://github.com/openpgpjs/openpgpjs/commit/b8f353abe83328824b76953a88696b854f13c5f9 via npm.